### PR TITLE
refactor!: move settings to apps sub-directories

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR moves the current settings to their own directory. It also adds a volume called production.py, allowing us to override production environment settings.

### How to test
Generate your environment dev|local, it should work the same. For the settings override, add to your config.yml:
```
ONTASK_EXTRA_PRODUCTION_SETTINGS = |
"""
LOGGING = # Login override...
"""
```

### CODE REVIEW

File: tutorontask/templates/ontask/apps/service/settings/base.py [hunk 1]
```diff
@@ -480,6 +493,42 @@
 }
 TINYMCE_COMPRESSOR = False
 
+# django-cors-headers
```
Comment (BryanttV) [inline tutorontask/templates/ontask/apps/service/settings/base.py:496]: Why is this section commented?

Comment (mariajgrimaldi) [reply tutorontask/templates/ontask/apps/service/settings/base.py:496]: That's how the file comes. No idea: https://github.com/abelardopardo/ontask_b/blob/b69f88d50a85a5a3addfd62b1462ca54249f766f/settings/base.py#L494-L528

**Review by BryanttV — APPROVED** [review]

File: tutorontask/templates/ontask/apps/service/settings/production.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+"""Configuration for production."""
+import logging.config
+
+from tutorontask.templates.ontask.apps.service.base import *  # NOQA
```
Comment (mariajgrimaldi) [inline tutorontask/templates/ontask/apps/service/settings/production.py]: Remove this

Comment (mariajgrimaldi) [reply tutorontask/templates/ontask/apps/service/settings/production.py]: Replace with:
```suggestion
from settings.base import *  # NOQA
```

**Pull request merged**